options/locale/locale_en-US.ini

```diff
@@ -421,6 +421,7 @@ remember_me.compromised = The login token is not valid anymore which may indicat
 forgot_password_title= Forgot Password
 forgot_password = Forgot password?
 need_account = Need an account?
+sign_up_tip = You are registering the first account in the system. Please carefully remember your username and password, as losing this information could cause significant inconvenience later.
```

Comment:
It is an unclear message. For example: why the "first account" is special? What "significant inconvenience" would happen? If it is not clean, it only confuses or frightens end users.